app/controllers/registrations_controller.rb

```diff
@@ -75,13 +88,15 @@ def check_allowed_email(resource)
     resource.errors.add(:email, "is not included in allowed domains.")
   end
 
-  def resolve_profile_field_issues
-    # Run this data update script when in a state of "first user" in the event
-    # that we are in a state where this was not already run.
-    # This is likely only temporarily needed.
-    return unless Settings::General.waiting_on_first_user
-
-    csv = Rails.root.join("lib/data/dev_profile_fields.csv")
-    ProfileFields::ImportFromCsv.call(csv)
+  def build_devise_resource
+    build_resource(sign_up_params)
+    resource.saw_onboarding = false
+    resource.registered = true
+    resource.registered_at = Time.current
+    resource.editor_version = "v2"
```

Comment:
Again: this defaults to `v1` in the database, then we override it here. Is it maybe time to update the default?

Reply:
I can address this in a different PR. The same change will need to be done in `authenticator.rb|`